### ISSUE TITLE
bindDn define are different in ldap and AD

### ISSUE THREAD

**Issue by SydOps** [puppet-community/puppet-rundeck#84]
Hi

I found the define for bindDn are different between `templates/_auth_ad.erb` and  `templates/_auth_ldap.erb`

```
templates/_auth_ad.erb

bindDn="<%= @auth_config['active_directory']['bind_dn']%>"

templates/_auth_ldap.erb
bindDn="CN=<%= @auth_config['ldap']['bind_dn']%>,<%= @auth_config['ldap']['user_base_dn'] %>"
```
Because of this, I got very strange result in ldpa setting (merged bindDN and userDN), and our ldap tree is not start from `CN=` . Fix is simple, but any reason to do that?

    bindDn="<%= @auth_config['ldap']['bind_dn']%>"

**Comment by liamjbennett** [comment]
Fixed in 57dd073

**Comment by rooty0** [comment]
hey @liamjbennett 
I don't really understand how this fix https://github.com/puppet-community/puppet-rundeck/commit/57dd07315844a499e832c26ebb3b60d228828ab1 going to resolve this issue. 

Official documentation:
```
....
bindDn="cn=Manager,dc=example,dc=com"
...
userBaseDn="ou=People,dc=test1,dc=example,dc=com"
``` 
where **bindDn** and **userBaseDn** completely different values
so I don't understand why do we have **userBaseDn** inside **bindDn**. Can we just remove **userBaseDn** from **bindDn** to make it more flexible?
Please check this fix https://github.com/puppet-community/puppet-rundeck/issues/96
